lib/dust.js

```diff
@@ -897,14 +856,6 @@
     }
   };
 
-  if (typeof define === "function" && define.amd && define.amd.dust === true) {
-    define("dust.core", function() {
-      return dust;
-    });
-  } else if (typeof exports === 'object') {
-    module.exports = dust;
-  } else {
-    root.dust = dust;
-  }
+  return dust;
 
-})((function(){return this;})());
```

Comment:
Yep, it should be 100% fixed now. I explain above why we were doing this-- it made things work by getting a windowlike, but we just shouldn't have used `root.console` to begin with because `root` should only be for the browser.